src/main/java/org/cbioportal/web/columnar/StudyViewColumnStoreController.java

```diff
@@ -371,6 +380,48 @@ public ResponseEntity<ClinicalViolinPlotData> fetchClinicalDataViolinPlots(
         return new ResponseEntity<>(result, HttpStatus.OK);
     }
 
+    // TODO move this to a utility class?
```

Comment:
This is mostly duplicate of https://github.com/cBioPortal/cbioportal/blob/956fb3379fb4d03559da1ecb355b0ee711fc0693/src/main/java/org/cbioportal/web/StudyViewController.java#L821-L850